docs/layers/identity/aws-sso.mdx

```diff
@@ -46,22 +46,32 @@ It's important to note that the specifics of the integration process may be subj
 
 ## Prerequisites
 
+First, enable the AWS IAM Identity Center (successor to AWS Single Sign-On) service in the `core-root` account. This is the account where the `aws-sso` component will be deployed.
+
 <Steps>
-  1. Go to the root admin account
-  2. Select primary region
-  3. Go to AWS SSO
-  4. Enable AWS SSO
+  1. Navigate to the `core-root` account in the AWS Web Console
+  2. Select your primary region
+  3. Go to AWS IAM Identity Center (successor to AWS Single Sign-On)
+  4. Enable the service
 </Steps>
 
 
-## Common Identity Providers
+## Configure your Identity Provider
 
-These are the instructions for the most common Identity Providers. For other providers, see the next section.
+These are the instructions for the most common Identity Providers. Alternatives are available, but the steps may vary depending on the provider.
 
 It's important to note that the specifics of the integration process may be subject to updates or changes, so it's recommended to refer to the official AWS documentation and respective IdP documentation for the most accurate and up-to-date information based on your current date.
 
+For providers not included in the following section, please [follow the AWS documentation for setting up an IdP integration with AWS](https://docs.aws.amazon.com/singlesignon/latest/userguide/supported-idps.html). This list includes Azure AD, CyberArk, OneLogin, and Ping Identity.
+
+:::caution Integrating JumpCloud with AWS IAM Identity Center
+
+The official AWS documentation for setting up JumpCloud with AWS IAM Identity Center is not accurate. Instead, please
+refer to the [JumpCloud official documentation](https://jumpcloud.com/support/integrate-with-aws-iam-identity-center)
+
+:::
 
```

Comment:
I think we should move this to a Jumpcloud tab.

Reply:
moved